src/main.py

```diff
@@ -1,60 +1,79 @@
 from flask import Flask, jsonify, request
-from flask_pymongo import PyMongo
+from pymongo import MongoClient
+from form_collection import FormCollection
 
 
 app = Flask(__name__)
 
+form_collection = FormCollection()
 
-#Running mongoDB on local host. Requires mongoDB to be installed and run locally. Must be generalized/dockerized
-app.config['MONGO_DBNAME'] = 'form'
-app.config['MONGO_URI'] = 'mongodb://localhost:27017/form'
 
-mongo = PyMongo(app)
+@app.route('/', methods=['GET'])
+def get_all_forms():
+    forms = form_collection.get_all_forms()
 
+    for form in forms:
+        del form['_id']
 
+    return jsonify({'all_forms' : forms})
 
-@app.route('/')
-def hello_docker():
-    return 'Hello Docker friends!'
 
-@app.route('/form', methods=['GET'])
-def get_all_forms():
-    form = mongo.db.form 
+@app.route('/<int:id>', methods=['GET'])
+def get_one_form(id):
+    
+    if id<0:
+        return jsonify({'message': 'invalid id'}), 401
```

Comment:
Don't need to jsonify the error message

Reply:
Actually, this would make sense, but we would have to agree on a common format, and change the other services... So I guess it's simpler to just stick to returning a string only